src/internal/m365/collection/groups/conversation_handler.go

```diff
@@ -175,6 +176,26 @@ func (bh conversationsBackupHandler) supportsItemMetadata() bool {
 	return true
 }
 
+func (bh conversationsBackupHandler) makeTombstones(
+	dps deltaPath.DeltaPaths,
+) (map[string]string, error) {
+	r := make(map[string]string, len(dps))
+
+	for id, v := range dps {
+		// ID is of format conversationID/threadID. Tombstones are looked up
+		// by conversationID only, so remove the threadID part. This is safe
+		// because every conversation has only one thread.
```

Comment:
Errant thought: we should add in a log or alert or something if we ever get more than one thread.